### DISCUSSION TITLE
Regarding Behaviour of intellij copilot chat plugin

### DISCUSSION THREAD

**Discussion by DaKiLloTh** [community/community#79248] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When performing a shift-return key combo (common key combination for inserting a new line instead of submitting form) intellij freezes for a moment then appears to insert a large amount of new lines.

intellij version. Build #IU-233.11799.300, built on December 12, 2023
copilot version. 1.4.5.4054-nightly

**Comment by levilansing** [comment]
I  have this problem as well. It seems the longer I wait before trying to click away, the longer it takes to respond. this time it inserted 4000+ lines and took several minutes to respond. it's extremely frustrating given how frequently I want to provide multi-line code samples in my questions

**Comment by usiohc** [comment]
The same bug happens in PyCharm

**Comment by jorourke** [comment]
How has this gone on for this long?

**Comment by JOlmstead495** [comment]
I am seeing this issue as well.
